src/app/content/highlights/components/Confirmation.tsx

```diff
@@ -18,7 +18,9 @@ export const Overlay = styled.div`
   ${cardBorder}
   transition: background 200ms;
   position: absolute;
-  display: flex;
+  ${({ isOpen }: { isOpen: boolean}) => {
```

Comment:
```suggestion
display: ${(props: {isOpen: boolean}) => props.isOpen ? 'flex' : 'none'};
```

Reply:
![image](https://user-images.githubusercontent.com/25258810/83375093-03ef0880-a38b-11ea-84cb-d18b1637877d.png)

z-indexes might be irregular